templates/homepage.html

```
@@ -220,9 +221,12 @@ <h3 class="pb-2 mb-4 text-lg md:text-2xl capitalize border-b border-gray-400 tex
         </div>
         <div class="flex flex-col justify-center items-center h-full pb-4">
           <div class="mx-auto">
-            <div class="flex items-center justify-center h-full py-4 rounded-lg">
-              <a href="https://www.youtube.com/watch?v=XB3uEit5R_Y" target="_blank" rel="noopener noreferrer" class="text-sm md:text-lg text-center text-sky-600 dark:text-sky-300 hover:text-orange dark:hover:text-orange">
-                Getting Started with Boost.Asio: Timers and Serial Ports
+            <div class="flex flex-col items-center justify-center h-full py-4 rounded-lg">
+              <a style href="https://youtu.be/myQRC4f9jTE" target="_blank" rel="noopener noreferrer" class="text-sm md:text-lg text-center text-sky-600 dark:text-sky-300 hover:text-orange dark:hover:text-orange">
+                The Open-Source Code Running On Mars - Boost Documentary Trailer
+              </a>
+              <a href='{% large_static 'img/v2/boost-trailer-thumbnail.png' %}'>
```

HTMLHint (1.9.2)

[error] Special characters must be escaped : [ < ].

(spec-char-escape)